01-node-tutorial/answers/12-https.js

```diff
@@ -0,0 +1,26 @@
+// const http = require('http');
+
+// const server = http.createServer((req, res) => {
+//   if (req.url === '/') {
+//     res.end('Welcome to our home page');
+//   }
+//   if (req.url === '/about') {
+//     res.end('Here is our short history');
+//   }
+//   res.end(`
+//   <h1>Oops! <h1>
+//   <p>We can't seem to find the page you are looking for </p>
+//   <a href="/">back home </a>`);
+// });
+
+// server.listen(5000);
+
+const http = require('http');
+
+const server = http.createServer((req, res) => {
+  if (req.url === '/') {
```

Comment:
Requires change:
If the URL isn’t a “/“, then I believe the page will hang. I’d recommend providing a fallback:
```js
if (req.url === '/') {
    return res.end('This is the homepage');
}

res.end('Page not found');
// return res.end('Page not found');
```
You must add a `return` to the first condition because it prevents an error where node will try to invoke `res.end()` twice (once in the if condition and again below). Alternatively, you could use an `if/else` statement.